client_cli/src/main.rs

```diff
@@ -115,41 +125,58 @@ fn main() -> Result<()> {
         config: Configuration(config),
         subcommand,
     } = Args::from_args();
-
-    println!("Value for config: {:?}", &config);
-    subcommand
-        .run(&config)
-        .wrap_err("Failed to run subcommand")?;
+    println!("Iroha Client CLI: build v0.0.1 [release]");
+    println!(
+        "Logged in as {}@{}",
```

Comment:
```suggestion
        "User: {}@{}",
```
`Log in` terminology I think does not apply to blockchains, this might confuse users.